.pre-commit-config.yaml

```diff
@@ -15,11 +15,6 @@ repos:
   - id: check-executables-have-shebangs
   - id: check-toml
 
-- repo: https://github.com/psf/black
-  rev: 23.1.0
-  hooks:
-  - id: black
```

Comment:
Just wanted to know why you want to remove `black`?

Reply:
We are able to enforce the double quotes from ruff. https://github.com/tbhaxor/AstroPlasma/blob/improve/logscale/pyproject.toml#L33

Reply:
Anyways I have added the black again for now